fdbserver/workloads/BlobRestoreWorkload.actor.cpp

```diff
@@ -72,7 +73,7 @@ struct BlobRestoreWorkload : TestWorkload {
 
 		if (self->performRestore_) {
 			fmt::print("Perform blob restore\n");
-			wait(store(result, self->extraDb_->blobRestore(normalKeys)));
+			wait(store(result, self->extraDb_->blobRestore(normalKeys, {})));
```

Comment:
Looks like we're not actually testing restoring to a previous version.
Is that something you wanted to do in a follow-on PR with this PR just being the plumbing for it? Or is this PR incomplete?

Reply:
I'll create a follow-up PR for tests. I did manual test with fdbcli and it passed.